apps/condo/domains/ticket/schema/CreateResidentTicketService.js

```diff
@@ -3,7 +3,7 @@
  */
 const { Contact } = require('@condo/domains/contact/utils/serverSchema')
 const { Property } = require('@condo/domains/property/utils/serverSchema')
-const { Ticket } = require('../utils/serverSchema')
+const { Ticket, TicketSource } = require('../utils/serverSchema')
```

Comment:
@domains

Reply:
Do we use domain import only when we import something from other domains and not from the same one?